### PR TITLE
added pipeline diagram

### PR DESCRIPTION
**Pull request by de-code**
Added a diagram giving an overview of the pipeline. Unfortunately GitHub doesn't seem to support _mermaid_ yet.

### CODE REVIEW

**Comment by seanwiseman** [comment]
Using the `atom` text editor, you can open an `.mmd` file and preview it and you can export it as a `.png` or `.svg`. You could then display the diagram in the readme.

**Comment by seanwiseman** [comment]
I used it in https://github.com/libero/walking-skeleton/pull/43

**Comment by de-code** [comment]
Okay, I added a PNG now.

File: pipeline.mmd [hunk 1]
```diff
@@ -0,0 +1,19 @@
+graph TD
+  subgraph Airflow
+    subgraph Collect and convert to CSV
+      ejp-xml["eJP XML"] --> csv-generator["CSV generator"]
+      ejp-csv["eJP CSV"] --> csv-generator
+      observer["Observer"] --> csv-generator
```
Comment (lsh-0) [inline pipeline.mmd]: this is interesting to me, as Observer is a purely derived data source

Comment (de-code) [reply pipeline.mmd]: It was originally planned for the initial prototype. But to fully replicate the editorial reports we need the publication dates which seemed simple enough to get out of observer (that's where the current reports get it from). We could consider going to the underlying data source in the future.

Comment (lsh-0) [reply pipeline.mmd]: Observer is a bit of a second class citizen at elife. it doesn't enjoy a CDN and doesn't have an API for access consistent with the one used by the rest of the infrastructure. The RSS feeds on the journal are linked directly I believe. Let me know before you start enumerating Observer results, it's never really experienced any load.

I don't see Observer going away, but chaining the scraping so you're scraping a scrape seems like chinese whispers.

**Pull request merged**